docs-src/ref-programs-compute.scrbl

```diff
@@ -394,6 +394,10 @@ Reach's @deftech{type}s are represented in programs by the following identifiers
   Bytes of different lengths are not compatible; however the shorter bytes may be @tech{padded}.}
   @item{@(mint-define! '("Digest")) @reachin{Digest}, which denotes a @tech{digest}.}
   @item{@(mint-define! '("Address")) @reachin{Address}, which denotes an @tech{account} @tech{address}.}
+  @item{
+    @margin-note{Reach has different representations of @tech{contract}s across @tech{connector}s.
+      For example, on Algorand a @reachin{Contract} is an Application ID, but on Ethereum it is an Address.}
+    @(mint-define! '("Contract")) @reachin{Contract}, which denotes the connection information for a @tech{contract}.}
```

Comment:
I wouldn't say "connection information". I dislike the word "connection" and would just leave it at "information" or maybe "identifying information"